### PR TITLE
add balanced flow rho theta

### PR DESCRIPTION
**Pull request by szy21**
Co-authored-by: @jiahe23 
This PR adds the balanced flow rho theta test case. It also changes the unicode in the filenames.

Still needs:
- [x] Change ci

cc @charleskawczynski @simonbyrne

### CODE REVIEW

File: examples/3dsphere/driver.jl [hunk 1]
```diff
@@ -9,7 +14,7 @@ using OrdinaryDiffEq
 
 const FT = Float64
 
-default_test_name = "baroclinic_wave_ρe"
+default_test_name = "balanced_flow_ρθ"
```
Comment (charleskawczynski) [inline examples/3dsphere/driver.jl:17]:
```suggestion
default_test_name = "balanced_flow_rhotheta"
```

File: examples/sphere/shallow_water.jl [hunk 2]
```diff
@@ -398,15 +398,15 @@ parameters = (; f = f, h_s = h_s)
...
+dt = 30 * 60
+T = 86400 * 1200
 
 prob = ODEProblem(rhs!, Y, (0.0, T), parameters)
 integrator = OrdinaryDiffEq.init(
     prob,
     SSPRK33(),
     dt = dt,
-    saveat = dt,
+    saveat = dt * 48,
```
Comment (charleskawczynski) [inline examples/sphere/shallow_water.jl:409]: ❤️

**Comment by szy21** [comment]
bors try

**Comment by szy21** [comment]
bors try

**Comment by charleskawczynski** [comment]
> @charleskawczynski should I just keep output at the last time step for this example?

When making animations animations, I'd recommend trying to aim for saving 50 solutions--and no more than 100. This is a heuristic, though, it's helpful to see what the animation looks like for different `saveat`s, and you basically want to minimize the number of saved solutions since they're expensive.

**Comment by szy21** [comment]
> > @charleskawczynski should I just keep output at the last time step for this example?
> 
> When making animations animations, I'd recommend trying to aim for saving 50 solutions--and no more than 100. This is a heuristic, though, it's helpful to see what the animation looks like for different `saveat`s, and you basically want to minimize the number of saved solutions since they're expensive.

Ok I changed it. If it looks good to you I will go ahead and bors r+:)

**Review by charleskawczynski — APPROVED** [review]

**Pull request merged**